lib/compress/clevels.h

```diff
@@ -22,110 +22,110 @@
 __attribute__((__unused__))
 #endif
 
-static const ZSTD_compressionParameters ZSTD_defaultCParameters[4][ZSTD_MAX_CLEVEL+1] = {
+static const ZSTD_CParams ZSTD_defaultCParameters[4][ZSTD_MAX_CLEVEL+1] = {
```

Comment:
Another way to pass this new parameter is to employ the existing `CCtxParams` structure,
which is internal, can be updated anytime, 
and is already used by many other advanced parameters,
like for example the recent `ZSTD_c_blockSplitterLevel`.